app/controllers/monthly_reports_controller.rb

```diff
@@ -1,6 +1,7 @@
 class MonthlyReportsController < ApplicationController
   def index
-    @monthly_reports = MonthlyReport.page params[:page]
+    users = User.where("name LIKE ?", "%#{params[:name]}%")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)